questionnaire/routes.js

```diff
@@ -118,8 +118,8 @@ router.route('/submission/confirm').post(async (req, res, next) => {
         }
         const err = Error(`The service is currently unavailable`);
```

Comment:
Can this error message be more explicit? I can't remember what we said it was catching